src/pages/Login/index.tsx

```diff
@@ -38,6 +38,14 @@ export function Login() {
       dispatch(loginUser({ email, password }));
     }
   };
+
+  const handleTestLogin = () => {
+    const testEmail = 'vineet@gmail.com';
+    const testPassword = 'vineet123';
+    setEmail(testEmail);
+    setPassword(testPassword);
```

Comment:
You can directly assign value in the setFunction instead of creating separate variables